### PR TITLE
Feature: support logs for evm hooks

### PR DESCRIPTION
**Pull request by thomas-nguy**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

- Needs to update ethermint to latest main https://github.com/tharsis/ethermint/commit/56c4a31f1055157565a391348234d484c981864c

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Review by yihuang — COMMENTED** [review]
Changes looks good, should we do some test on it?

**Comment by yihuang** [comment]
> for the integration tests, we need the latest ethermint main.
> 
> Is it safe to upgrade ethermint at this moment? or still need to work on a forked version for the final release?

time to create the `release/v0.7.x` branch I guess.

**Review by linfeng-crypto — APPROVED** [review]

**Comment by thomas-nguy** [comment]
blocked by https://github.com/tharsis/ethermint/pull/1088

File: go.mod [hunk 1]
```diff
@@ -153,7 +153,7 @@ require (
 replace (
 	// TODO: fix keyring upstream
 	github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.4
+	github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.4-0.20220518050709-bd4ca739c699
```
Comment (thomas-nguy) [inline go.mod:156]: needed for ethermint main dependency

File: integration_tests/test_gravity.py [hunk 2]
```diff
@@ -236,6 +246,13 @@ def check_gravity_native_tokens():
             ADDRS["validator"], amount, 0
         ).buildTransaction({"from": ADDRS["community"]})
         txreceipt = send_transaction(cronos_w3, tx, KEYS["community"])
+        # CRC20 emit 3 logs for send_to_ethereum:
+        # burn
+        # __CronosSendToEthereum
+        # __CronosSendToEthereumResponse
+        assert len(txreceipt.logs) == 3
+        assert get_id_from_receipt(txreceipt) == \
```
Comment (thomas-nguy) [inline integration_tests/test_gravity.py:254]: added a test

File: go.mod [hunk 3]
```diff
@@ -17,11 +17,11 @@ require (
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.1
-	github.com/tendermint/tendermint v0.34.19
+	github.com/tendermint/tendermint v0.34.20-0.20220517115723-e6f071164839
 	github.com/tendermint/tm-db v0.6.7
-	github.com/tharsis/ethermint v0.6.1-0.20220503175102-93d15db4d086
+	github.com/tharsis/ethermint v0.6.1-0.20220519101126-9edc87d1147f
```
Comment (thomas-nguy) [inline go.mod:22]: update ethermint to latest

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**